src/account/controllers/image.controller.ts

```diff
@@ -0,0 +1,98 @@
+import {
+    Controller,
+    Post,
+    UploadedFile,
+    UploadedFiles,
+    UseGuards,
+    UseInterceptors,
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { FileFieldsInterceptor, FileInterceptor } from '@nestjs/platform-express';
+import { ApiBody, ApiConsumes, ApiCreatedResponse, ApiTags } from '@nestjs/swagger';
+import { AssociatedImage } from '../models/associated-project.model';
+import { ImageUploadService } from '../services/image-upload.service';
+import { UpdateProjectService } from '../services/update-project.service';
+
+@ApiTags('account')
+@UseGuards(AuthGuard('jwt-refresh'))
+@Controller('account')
+export class ImageController {
+    public constructor(private imageUploadService: ImageUploadService) {}
+
+    @Post('logos')
```

Comment:
Do we need to pluralize this endpoint? Only one logo can be set at the same time.

Reply:
The REST naming convention states that collections should be reverenced as plurals https://restfulapi.net/resource-naming/